### PR TITLE
scan env for HTTP_GATEWAY

### PR DESCRIPTION
**Pull request by jldec**
updated server to scan environment for `*_HTTP_GATEWAY_SERVICE_HOST`
this should make it work independent of helm name used

also turned off verbose logging of the vote POSTs

### CODE REVIEW

File: ui/README.md [hunk 1]
```diff
@@ -4,9 +4,11 @@ nodejs socket.io server backed by redis
 The server looks for redis using environment variables `REDIS_HOST`, `REDIS_PORT`, `REDIS_PASSWORD`.
 Defaults are localhost, 6379, and no auth.
 
+Current votecounts are displayed by monitoring the `demo:votes` hash in redis.
+
 Votes are posted to the votes topic in Riff via the http-gateway service configured 
-at `HTTP_GATEWAY_SERVICE_HOST` and `HTTP_GATEWAY_SERVICE_PORT`. Current votecounts are
-displayed by monitoring the `demo:votes` hash in redis.
+at `HTTP_GATEWAY_SERVICE_HOST` and `HTTP_GATEWAY_SERVICE_PORT` (otionally prefixed with
```
Comment (markfisher) [inline ui/README.md]: typo: "otionally"

File: ui/scripts/localenv [hunk 2]
```diff
@@ -4,6 +4,6 @@ export REDIS_HOST=$(minikube ip)
 export REDIS_PASSWORD=$(kubectl get secret --namespace default counters-redis -o jsonpath="{.data.redis-password}" | base64 --decode)
 
 export HTTP_GATEWAY_SERVICE_HOST=$(minikube ip)
-export HTTP_GATEWAY_SERVICE_PORT=$(kubectl get svc http-gateway -o jsonpath='{.spec.ports[0].nodePort}')
+export HTTP_GATEWAY_SERVICE_PORT=$(kubectl get svc demo-riff-http-gateway -o jsonpath='{.spec.ports[0].nodePort}')
```
Comment (markfisher) [inline ui/scripts/localenv]: can use `get svc -l component=http-gateway` instead of the name

Comment (trisberg) [reply ui/scripts/localenv]: This is what we use in request/publish scripts for NodePort -  `$(kubectl get svc -l component=http-gateway -o jsonpath='{.items[0].spec.ports[?(@.name == "http")].nodePort}')`

**Comment by jldec** [comment]
updated with suggested fixes.

**Comment by markfisher** [comment]
LGTM; squashed and merged as https://github.com/markfisher/s1p2017-faas-demo/commit/7fd2de411f1a5b360f8f980ce231786dee428b5e

**Pull request closed without merging**